cmd/admin-heal.go

```diff
@@ -96,20 +96,36 @@ var adminHealCmd = cli.Command{
 
 // healMessage container to hold repair information.
 type healMessage struct {
-	Status string             `json:"status"`
-	Bucket string             `json:"bucket"`
-	Object *madmin.ObjectInfo `json:"object"`
-	Upload *madmin.UploadInfo `json:"upload"`
+	Status string                  `json:"status"`
+	Result madmin.HealObjectResult `json:"result"`
```

Comment:
@vadmeste I meant `madmin.HealObjectResult `. I agree that the name suggests that it returns result of the healing of objects and it can be confusing when used with result of healing uploads. I have named the member variable `Result` to make it applicable for both objects and uploads and left the type as-is. Do you have a different approach in mind?